### PR TITLE
Sync up spark2 explain code

### PR DESCRIPTION
**Pull request by GaryShen2008**
based on commit id 2004768fc

Signed-off-by: Gary Shen <gashen@nvidia.com>

Contribute to https://github.com/NVIDIA/spark-rapids/issues/7139

### CODE REVIEW

File: scripts/spark2diffs/RegexParser.diff [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <  * Copyright (c) 2022, NVIDIA CORPORATION.
 ---
 >  * Copyright (c) 2021-2022, NVIDIA CORPORATION.
```
Comment (tgravescs) [inline scripts/spark2diffs/RegexParser.diff:4]: this shouldn't have changed

File: scripts/spark2diffs/RapidsConf.diff [hunk 2]
```diff
@@ -5,23 +5,23 @@
 23a24,25
 > import ai.rapids.cudf.Cuda
 > 
-321c323
+333c335
 <     .createWithDefault(ByteUnit.GiB.toBytes(1).toLong)
 ---
 >     .createWithDefault(ByteUnit.GiB.toBytes(1))
```
Comment (tgravescs) [inline scripts/spark2diffs/RapidsConf.diff:11]: these  shouldn't have changed, does this build?

**Review by tgravescs — COMMENTED** [review]
based on the one change in RapidsConf I don't believe that this would run.   Has this been built and tested yet?

File: scripts/spark2diffs/RegexParser.diff (hunk 1 above)
Comment (GaryShen2008) [reply scripts/spark2diffs/RegexParser.diff:4]: Yeah, it's strange that it shows a difference here.
I compared locally and it only shows the line5.

Comment (GaryShen2008) [reply scripts/spark2diffs/RegexParser.diff:4]: I think it just shows the content up and down around the changed place.
The content contains the similar format of a diff file, then github shows it as a difference.
But there's no +/- mark in the lines.

File: scripts/spark2diffs/RapidsConf.diff (hunk 2 above)
Comment (GaryShen2008) [reply scripts/spark2diffs/RapidsConf.diff:11]: Same as previous comments. Github shows the lines around the changed line.

**Comment by GaryShen2008** [comment]
yes, I built locally and tested on egx cluster.

**Review by tgravescs — COMMENTED** [review]
the diff seems to have a bunch of things on the base branch or perhaps a merge went wrong here?  Like configs.md, supported_ops.md, array_test.py, etc...

**Review by tgravescs — COMMENTED** [review]
mostly looks good, other then the diff and files that don't seem like that hsould be in here.

**Comment by GaryShen2008** [comment]
build

**Comment by GaryShen2008** [comment]
HI @tgravescs, can you take another review this PR?

**Review by tgravescs — APPROVED** [review]

**Pull request merged**